<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportPdfDialog</class>
 <widget class="QDialog" name="ExportPdfDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>680</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>462</width>
    <height>546</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Export PDF Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>778</width>
        <height>617</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout" columnstretch="1,1">
       <item row="0" column="0" rowspan="2">
        <widget class="QGroupBox" name="groupSize">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set the PDF page (paper) size.  &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="title">
          <string>Page Size</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="vLayoutSize">
            <item>
             <widget class="QRadioButton" name="radioButtonA6">
              <property name="text">
               <string>A6 (105 x 148 mm)</string>
              </property>
              <property name="_settings_value" stdset="0">
               <UInt>0</UInt>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroupPaperSize</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButtonA5">
              <property name="text">
               <string>A5 (148 x 210 mm)</string>
              </property>
              <property name="_settings_value" stdset="0">
               <UInt>1</UInt>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroupPaperSize</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButtonA4">
              <property name="text">
               <string>A4 (210x297 mm)</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <property name="_settings_value" stdset="0">
               <UInt>2</UInt>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroupPaperSize</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButtonA3">
              <property name="text">
               <string>A3 (297x420 mm)</string>
              </property>
              <property name="_settings_value" stdset="0">
               <UInt>3</UInt>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroupPaperSize</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButtonLetter">
              <property name="text">
               <string>Letter (8.5x11 in)</string>
              </property>
              <property name="_settings_value" stdset="0">
               <UInt>4</UInt>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroupPaperSize</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButtonLegal">
              <property name="text">
               <string>Legal (8.5x14 in)</string>
              </property>
              <property name="_settings_value" stdset="0">
               <UInt>5</UInt>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroupPaperSize</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButtonTabloid">
              <property name="text">
               <string>Tabloid (11x17 in)</string>
              </property>
              <property name="Tabloid" stdset="0">
               <string/>
              </property>
              <property name="_settings_value" stdset="0">
               <UInt>6</UInt>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroupPaperSize</string>
              </attribute>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QGroupBox" name="groupMetaData">
         <property name="title">
          <string>Meta data</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>The fields Title, Creator and CreateDate are always filled in the exported file when meta data is enabled.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayoutMetaData">
            <property name="horizontalSpacing">
             <number>16</number>
            </property>
            <property name="verticalSpacing">
             <number>8</number>
            </property>
            <item row="3" column="0">
             <widget class="QCheckBox" name="checkBoxMetaDataSubject">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="toolTip">
               <string>A description of the document</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEditMetaDataTitle">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="placeholderText">
               <string>Title, leave empty to use the file name</string>
              </property>
              <property name="clearButtonEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="lineEditMetaDataSubject">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="placeholderText">
               <string>Subject</string>
              </property>
              <property name="clearButtonEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="lineEditMetaDataKeywords">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="toolTip">
               <string>A copyright associated with this document</string>
              </property>
              <property name="placeholderText">
               <string>Keywords</string>
              </property>
              <property name="clearButtonEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QCheckBox" name="checkBoxMetaDataAuthor">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="toolTip">
               <string>A name for a designer of this document</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="lineEditMetaDataAuthor">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="placeholderText">
               <string>Author</string>
              </property>
              <property name="clearButtonEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QCheckBox" name="checkBoxMetaDataKeywords">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupOrientation">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set the direction of the largest page dimension.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="title">
          <string>Page Orientation</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="vLayoutOrient">
            <item>
             <widget class="QRadioButton" name="rbOPort">
              <property name="text">
               <string>Portrait (Vertical)</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <property name="_settings_value" stdset="0">
               <UInt>1</UInt>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroupOrientation</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rbOLand">
              <property name="text">
               <string>Landscape (Horizontal)</string>
              </property>
              <property name="_settings_value" stdset="0">
               <UInt>2</UInt>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroupOrientation</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rbOAuto">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Determine best orientation based on maximum geometry dimension.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>Auto</string>
              </property>
              <property name="_settings_value" stdset="0">
               <UInt>0</UInt>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroupOrientation</string>
              </attribute>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QGroupBox" name="groupAnnotations">
         <property name="title">
          <string>Annotations</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="vLayoutAnnot">
            <item>
             <widget class="QCheckBox" name="checkBoxShowFilename">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Include design filename on page.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>Show Design Filename</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupScale">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Include rulers on page to confirm 1:1 printing scale.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="title">
               <string>Show Scale</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>6</number>
               </property>
               <item>
                <widget class="QCheckBox" name="checkBoxShowScaleMessage">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="toolTip">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Include text describing usage of scale.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="text">
                  <string>Show Scale Usage</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupGrid">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Include a grid of the selected size on the page.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="title">
               <string>Show Grid</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>6</number>
               </property>
               <item>
                <layout class="QHBoxLayout" name="hLayoutGrid">
                 <item>
                  <widget class="QRadioButton" name="rbGs_2mm">
                   <property name="text">
                    <string>2mm</string>
                   </property>
                   <property name="_selected_value" stdset="0">
                    <double>2.000000000000000</double>
                   </property>
                   <attribute name="buttonGroup">
                    <string notr="true">buttonGroupGridSize</string>
                   </attribute>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="rbGs_2r5mm">
                   <property name="text">
                    <string>2.5mm</string>
                   </property>
                   <property name="_selected_value" stdset="0">
                    <double>2.500000000000000</double>
                   </property>
                   <attribute name="buttonGroup">
                    <string notr="true">buttonGroupGridSize</string>
                   </attribute>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="rbGs_4mm">
                   <property name="text">
                    <string>4mm</string>
                   </property>
                   <property name="_selected_value" stdset="0">
                    <double>4.000000000000000</double>
                   </property>
                   <attribute name="buttonGroup">
                    <string notr="true">buttonGroupGridSize</string>
                   </attribute>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="rbGs_5mm">
                   <property name="text">
                    <string>5mm</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                   <property name="_selected_value" stdset="0">
                    <double>5.000000000000000</double>
                   </property>
                   <attribute name="buttonGroup">
                    <string notr="true">buttonGroupGridSize</string>
                   </attribute>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="rbGs_10mm">
                   <property name="text">
                    <string>10mm</string>
                   </property>
                   <property name="_selected_value" stdset="0">
                    <double>10.000000000000000</double>
                   </property>
                   <attribute name="buttonGroup">
                    <string notr="true">buttonGroupGridSize</string>
                   </attribute>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <spacer name="verticalSpacerPdfDialog">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutDialogButtons">
     <property name="topMargin">
      <number>9</number>
     </property>
     <item>
      <widget class="QCheckBox" name="checkBoxAlwaysShowDialog">
       <property name="text">
        <string>Always show dialog</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerDialogButtons">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOk">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pushButtonOk</tabstop>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>scrollArea</tabstop>
  <tabstop>radioButtonA6</tabstop>
  <tabstop>radioButtonA5</tabstop>
  <tabstop>radioButtonA4</tabstop>
  <tabstop>radioButtonA3</tabstop>
  <tabstop>radioButtonLetter</tabstop>
  <tabstop>radioButtonLegal</tabstop>
  <tabstop>radioButtonTabloid</tabstop>
  <tabstop>rbOPort</tabstop>
  <tabstop>rbOLand</tabstop>
  <tabstop>rbOAuto</tabstop>
  <tabstop>checkBoxShowFilename</tabstop>
  <tabstop>groupScale</tabstop>
  <tabstop>checkBoxShowScaleMessage</tabstop>
  <tabstop>groupGrid</tabstop>
  <tabstop>rbGs_2mm</tabstop>
  <tabstop>rbGs_2r5mm</tabstop>
  <tabstop>rbGs_4mm</tabstop>
  <tabstop>rbGs_5mm</tabstop>
  <tabstop>rbGs_10mm</tabstop>
  <tabstop>groupMetaData</tabstop>
  <tabstop>lineEditMetaDataTitle</tabstop>
  <tabstop>checkBoxMetaDataAuthor</tabstop>
  <tabstop>lineEditMetaDataAuthor</tabstop>
  <tabstop>checkBoxMetaDataSubject</tabstop>
  <tabstop>lineEditMetaDataSubject</tabstop>
  <tabstop>checkBoxMetaDataKeywords</tabstop>
  <tabstop>lineEditMetaDataKeywords</tabstop>
  <tabstop>checkBoxAlwaysShowDialog</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonOk</sender>
   <signal>clicked()</signal>
   <receiver>ExportPdfDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>484</x>
     <y>522</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>ExportPdfDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>522</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroupPaperSize"/>
  <buttongroup name="buttonGroupOrientation"/>
  <buttongroup name="buttonGroupGridSize"/>
 </buttongroups>
</ui>
